scripts/installRdeps.R

```diff
@@ -12,6 +12,7 @@
 .install_pkg("BiocManager")
 .install_pkg("optparse")
 .install_pkg("devtools")
+.install_pkg("conflicted")
```

Comment:
Do you think conflicted makes sense as a dependency here? I'm not sure we should push that on all downstream users...

Reply:
I just double checked, turns out we don't need 'conflicted'. I just removed it.